site/docs/spark-configuration.md

```diff
@@ -155,6 +155,7 @@ spark.read
 | file-open-cost  | As per table property | Overrides this table's read.split.open-file-cost                                          |
 | vectorization-enabled  | As per table property | Overrides this table's read.parquet.vectorization.enabled                                          |
 | batch-size  | As per table property | Overrides this table's read.parquet.vectorization.batch-size                                          |
+| stream-from-timestamp | Long.MIN_VALUE | Timestamp in milliseconds; start a stream from the snapshot that occurs after this timestamp. |
```

Comment:
Nit: Could you reword the description to be a bit more clear about which snapshot is started from? That was a source of confusion for a while for me.

Maybe `Timestamp in milliseconds, start streaming this table from the first snapshot that occurs strictly after this timestamp`?

It's a minor change, but it took me a while to be sure which snapshot was being referred to after being explained it, so I think it might help others as well to be more sure of what we mean.

Reply:
Nit: Also, the rest of the descriptions don't seem to end in periods. I'm not sure if that's because they end with configuration keys, but if other sentences in the same column don't end with periods can you remove it?

Reply:
Hi @kbendick 

Thanks for reviewing!
 
`stream-from-timestamp` would actually stream the table from the snapshot >= timestamp. Here's an example: https://github.com/apache/iceberg/blob/master/core/src/main/java/org/apache/iceberg/util/SnapshotUtil.java#L109-L117

Hence, I have kept the description as `Timestamp in milliseconds, start streaming this table from the first snapshot that occurs strictly after this timestamp`. 

Please let me know if we can phrase it better.

Reply:
Ah ok cool. I knew I got confused by that somehow so I appreciate you detailing it for me. If it’s greater than or equal, probably `the first snapshot that occurs at or after this timestamp` is fine. The strictly after implies strictly greater than. That was my mistake. I apologize.

Reply:
Makes sense! I have made the changes.